README.md

```diff
@@ -61,10 +61,123 @@ By default the following dependencies are enabled:
 - [Strimzi Kafka Operator](https://strimzi.io/)
 - [Cert Manager](https://github.com/cert-manager/cert-manager)
 
+
+
+### Deploy in EKS Cluster
+
+To deploy the Big Peer into a AWS EKS Cluster like EKS you will need to perform the following actions;
+
+#### EKS Cluster Setup Checks
+
+- Create the [EKSClusterRole](https://docs.aws.amazon.com/eks/latest/userguide/cluster-iam-role.html#create-service-role)
+
+
+- Install [eksctl](https://eksctl.io/installation/)
+
+- Deploy EKS cluster with eksctl command
+
+<br/>
+
+```shell
+eksctl create cluster --name <cluster-name> --region <aws-region> --profile <aws-profile>
+```
+<br/>
+
+- Once the cluster is up and running verify that you have [ebs-csi-driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html#managing-ebs-csi) running in the `kube-system` namespace. This ensures that you have the provisioner that will dynamicallly create the Persistent Volumes that the hydra-store and kafka zookeeper nodes will need.
+
+<br/>
+
+- If your default storageClass is not set as the the default storageClass you can do that by adding the line below to the annotations.
```

Comment:
I don't think it is clear where this should go?

Reply:
I included it as part of the things to note after you deploy the EKS cluster and before you deploy the chart. Would you want that taken out?